code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -0,0 +1,279 @@
+		stat(null, "Current True Name: [true_name]")
+		stat(null, "Host: [host ? host : "NONE"]")
+		stat(null, "You are [recovering ? "too weak" : "able"] to deploy!")
+
+/mob/living/simple_animal/hostile/clockwork_marauder/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, list/spans)
+	..()
+	if(findtext(message, true_name) && is_in_host()) //Called or revealed by hearing their true name
+		if(speaker == host)
+			emerge_from_host()
+		else
+			src << "<span class='warning'><b>You hear your true name and partially emerge before you can stop yourself!</b></span>"
+			host.visible_message("<span class='warning'>[host]'s skin flashes crimson!</span>", "<span class='warning'><b>Your marauder instinctively reacts to its true name!</b></span>")
+
+/mob/living/simple_animal/hostile/clockwork_marauder/say(message)
+	if(is_in_host())
+		src << "<span class='warning'>You cannot speak while inside of your host! Used the Linked Minds verb instead.</span>"
+		return 0
+	..()
+
+/mob/living/simple_animal/hostile/clockwork_marauder/adjustHealth(amount) //Fatigue damage
+	for(var/mob/living/L in range(3, src))
+		if(L.null_rod_check()) //Null rods allow direct damage
+			src << "<span class='userdanger'>The power of a holy artifact bypasses your armor and wounds you directly!</span>"
+			. = ..()
+			break
+			return 0
+	adjust_fatigue(amount)
+	return 1
+
+/mob/living/simple_animal/hostile/clockwork_marauder/AttackingTarget()
+	if(is_in_host())
+		return 0
+	..()
+
+/mob/living/simple_animal/hostile/clockwork_marauder/proc/adjust_fatigue(amount) //Adds or removes the given amount of fatigue
+	if(!ratvar_awakens)
+		fatigue = max(0, min(fatigue + amount, fatigue_recall_threshold))
+	return 1
+
+/mob/living/simple_animal/hostile/clockwork_marauder/verb/linked_minds() //Discreet communications between a marauder and its host
+	set name = "Linked Minds"
+	set desc = "Silently communicates with your master."
+	set category = "Marauder"
+
+	if(!host) //Verb isn't removed because they might gain one... somehow
+		usr << "<span class='warning'>You don't have a host!</span>"
+		return 0
+	var/message = stripped_input(usr, "Enter a message to tell your host.", "Telepathy")// as null|anything
+	if(!usr || !message)
+		return 0
+	if(!host)
+		usr << "<span class='warning'>Your host seems to have vanished!</span>"
+		return 0
+	message = "<span class='heavy_brass'>Marauder [true_name]:</span> <span class='brass'>\"[message]\"</span>" //Processed output
+	usr << message
+	host << message
+	return 1
+
+/mob/living/proc/talk_with_marauder() //See above - this is the host version
+	set name = "Linked Minds"
+	set desc = "Silently communicates with your marauder."
+	set category = "Clockwork"
+	var/mob/living/simple_animal/hostile/clockwork_marauder/marauder
+
+	if(!marauder)
+		for(var/mob/living/simple_animal/hostile/clockwork_marauder/C in living_mob_list)
+			if(C.host == src)
+				marauder = C
```

Comment:
what happens if you have multiple? you should either prevent multiple or make it send the message to all hosted marauders